### PR TITLE
Added Readme

### LINKED ISSUE #4: Update README
Update the README with all the necessary information required for building this project. The README must include at least the following

- Instruction to clone and setup the environment
- Instruction to build the project
- Relevant project badges
- Add relevant link

You may take a look at the README of other repositories for inspiration.

### PR DESCRIPTION
**Pull request by Monik09**
Fixes #4

### CODE REVIEW

**Review by thecodepapaya — CHANGES_REQUESTED** [review]
Could you also add a `Made With ♥ By IIITV Coding Club` footer as seen in [getmein-web](https://github.com/iiitv/getmein-web/) repository?

**Comment by thecodepapaya** [comment]
Also mention in README that the new contributors should go through [Wiki](https://github.com/iiitv/club_calender/wiki) first to get an idea about the project before starting to contribute.

File: README.md [hunk 1]
```diff
@@ -72,3 +71,14 @@ To run the program, follow the steps given below:
...
 <ul>
   <li><a href=https://flutter.dev/>Flutter </a> </li>
+</ul>
+
+## Contributing :raising_hand:
+Go through the [Wiki](https://github.com/iiitv/club_calender/wiki) to get an idea of the project before contributing to it.
+  
+</ul>
+
+<h1 align='center'>
```
Comment (thecodepapaya) [inline README.md:81]: Isn't the footer a bit too large?

**Review by thecodepapaya — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Monik09) [reply README.md:81]: ok I make it short

**Review by thecodepapaya — APPROVED** [review]

**Pull request merged**